plugins/disabled-ZeronameLocal/NamecoinUtils.py

```diff
@@ -0,0 +1,43 @@
+import os, signal, json, requests, subprocess, time
+from pathlib import Path
+from configparser import ConfigParser
+import psutil
+
+
+def is_namecoin_installed():
+    home = str(Path.home())
```

Comment:
Does `Path.home()` will work for macOS and Windows ? 
Windows XP path is `C:\Documents and Settings\<username>\Application Data\Namecoin\namecoin.conf`
Windows Vista 7, 10, 11? path is `C:\Users\<username>\AppData\Roaming\Namecoin`
macOS path is `/Users/<username>/Library/Application Support/Namecoin/namecoin.conf`

Reply:
Windows XP, Vista and 7 are no longer supported, are they plenty of users with theses windows versions ?

Reply:
@ethicnology do not hardcore any path.